griptape/tasks/base_text_input_task.py

```diff
@@ -14,14 +15,19 @@ class BaseTextInputTask(BaseTask, ABC):
     DEFAULT_RULESET_NAME = "Default Ruleset"
     ADDITIONAL_RULESET_NAME = "Additional Ruleset"
 
-    input_template: str = field(default=DEFAULT_INPUT_TEMPLATE)
+    input_template: str | TextArtifact | Callable[[BaseTask], TextArtifact] = field(default=DEFAULT_INPUT_TEMPLATE)
```

Comment:
Should this be renamed to `_input`?

Reply:
I think so. We're only rendering a template in one of the three input cases so that name seems overly specific anymore